app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
```

Comment:
How about a name like `ApplicationView` to make it clear what this is for and why it's not used on other models?

Reply:
See https://github.com/rails/rails/pull/22567. New rails 5 projects generate this by default.